packages/react-storage/src/components/StorageBrowser/subcomponents/BreadcrumbsControl/BreadcrumbsControl.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { NavElementProps } from '@aws-amplify/ui-react/internal';
+import { useElement } from '../../context/elements';
+import { Breadcrumb } from './Breadcrumb';
+import { Item } from './Item';
+import { List } from './List';
+import { Separator } from './Separator';
+
+const BreadcrumbsControlPrimitive = <T extends NavElementProps>({
```

Comment:
Let's not use "Primitive" in any of our naming, if this is meant to be the `Navigate` control then that's the name it should have